b1ddi/resource_ipam_address.go

```diff
@@ -27,14 +28,21 @@ func resourceIpamsvcAddress() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 		Schema: map[string]*schema.Schema{
-
+			// Defined for Terraform usage. This field won't be used in the API call
+			"next_available_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ExactlyOneOf: []string{"address", "next_available_id"},
+				Description:  "The resource ID in the form \"/ipam/[address_block|subnet|range]/<UUID>\".",
```

Comment:
The description can be more verbose in trying to explain that this will create the next available resource in the given container